configtx/orderer.go

```diff
@@ -821,6 +828,36 @@ func addOrdererValues(ordererGroup *cb.ConfigGroup, o Orderer) error {
 		if consensusMetadata, err = marshalEtcdRaftMetadata(o.EtcdRaft); err != nil {
 			return fmt.Errorf("marshaling etcdraft metadata for orderer type '%s': %v", orderer.ConsensusTypeEtcdRaft, err)
 		}
+	case orderer.ConsensusTypeBFT:
```

Comment:
There are many other places where `case orderer.ConsensusTypeEtcdRaft:` exists but `case orderer.ConsensusTypeBFT:` does not. For example: `func (o *OrdererGroup) Configuration() (Orderer, error)`

Please scan the project for any function that uses `case orderer.ConsensusTypeEtcdRaft:` and see whether BFT needs to be applied there as well.

Reply:
done